website/siteConfig.js

```diff
@@ -12,6 +12,14 @@
 // If on netlify, use '/'. Otherwise use '/KaTeX/'.
 const baseUrl = process.env.BASE_URL || (process.env.CONTEXT ? '/' : '/KaTeX/');
 
+// Plugin for Remarkable to inject variables
+const {Plugin: Embed} = require('remarkable-embed');
+const embed = new Embed();
```

Comment:
So this replaces `program.command('markdown-help')`?

Reply:
@kevinbarabash No, this is for the stylesheet inclusion.